src/4.3_to_5.0/__snapshots__/migrate_43_to_50.test.ts.snap

```diff
@@ -2404,86 +2397,6 @@ exports[`migrate_43_to_50 returns a valid ActiveUI5 /ui folder on a small input
       },
     },
     "users": {
-      "children": {
-        "admin": {
-          "children": {
-            "activity": {
-              "entry": {
-                "content": "{"userFilters":["[Geography].[City].[ALL].[AllMember].[Berlin]"]}",
-                "owners": [
-                  "admin",
-                ],
-                "readers": [
-                  "admin",
-                ],
-              },
-            },
-            "settings": {
-              "entry": {
-                "content": "{"theme":"dark-activeviam","search.maxResults":10,"userFilters.areEnabled":true,"drillthrough.defaultSelectedColumns":{"EquityDerivativesCube":["delta","gamma","pnlVega","Desk","Currency","Date","HostName"],"EquityDerivativesCubeDist":["BumpedMtmDown","ProductQtyMultiplier","vega","rho","productId","pnlVega","pnlDelta","pnl","gamma","delta","TradeId","ProductBaseMtm"]}}",
```

Comment:
Oops, it looks like we're losing the settings conversion part of this test snapshot. It's probably because some settings that were part of a legacy UI folder test resource were lost in the process. Can you please make sure that we don't lose this?

Reply:
I've added a new commit to avoid losing this information. However, it means re-adding the `settings` to the `emptyLegacyUIFolder` since we are now generating test resources such as `smallLegacyUIFolder` via the `emptyLegacyUIFolder` and `addLegacyBookmarkToUIFolder`.